src/API.h

```diff
@@ -0,0 +1,101 @@
+//
+//  API.h
```

Comment:
I think we should call this header `drafter.h` so people will include `drafter.h` and not `API.h` when using this library. This will be clearer and consistent with other C/C++ libraries.

Examples:

- Curl (`#include <curl/curl.h>`)
- gnutls (`#include <gnutls/gnutls.h>`)